lib/engine/config.ex

```diff
@@ -63,6 +74,13 @@ defmodule Engine.Config do
   def create_tables(state) do
     :ets.new(state.table_name_signs, [:set, :protected, :named_table, read_concurrency: true])
     Headways.create_table(state.table_name_headways)
+
+    :ets.new(state.table_name_chelsea_bridge, [
```

Comment:
Yeah, just consistency. Though I also wonder: Is the overhead of process calls substantial enough that we'd look to avoid it at the scale we're at, and that's why we seem to prefer ETS?